packages/wmr/src/plugins/wmr/client.js

```diff
@@ -31,6 +41,23 @@ function connect(needsReload) {
 
 	ws.addEventListener('message', handleMessage);
 	ws.addEventListener('error', handleError);
+	ws.addEventListener('close', reconnect);
+}
+
+function reconnect() {
```

Comment:
Currently reconnect only gets called once, should this be called again if establishing connection in `connect(false)` fails?